src/datreant/core/trees.py

```diff
@@ -467,3 +468,26 @@ def make(self):
         self.makedirs()
 
         return self
+
+    def sync(self, dest, mode='upload', compress=True, backup=False, dry=False,
```

Comment:
Would it be clearer to use `other` instead of `dest` since this works both in the "upload" and "download" sense?